services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_background_task.py

```diff
@@ -94,7 +95,9 @@ async def _close_unhealthy_service(
         return
 
     # 2. Close the billing transaction (as not billed)
-    if running_service.wallet_id and running_service.pricing_unit_cost:
+    if running_service.wallet_id:
+        assert isinstance(running_service.pricing_unit_cost, Decimal)  # nosec
```

Comment:
Is it not possible to add the type hint directly to `pricing_unit_cost` field?

Reply:
pricing_unit_cost is Decimal or None. Here I know that when wallet_id is provided this implies that pricing_unit_cost is Decimal.